src/retroshare/rsconfig.h

```diff
@@ -389,17 +389,30 @@ class RsServerConfig
 	 * @param[in] opModeStr new operating mode as string
 	 * @return
 	 */
-    virtual bool     setOperatingMode(const std::string &opModeStr) = 0;
+	virtual bool     setOperatingMode(const std::string &opModeStr) = 0;
```

Comment:
now that you touched it can you fix the indentation and spacing too? Just one tab before `virtual` just one space between `bool` and `setOperatingMode`

Reply:
Done